doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -822,14 +819,36 @@ the split because it has not changed.
   .. image:: ../_static/diffs-coqide-multigoal.png
      :alt: coqide with Set Diffs on with multiple goals
 
-This is how diffs may appear after applying a :tacn:`intro` tactic that results
-in compacted hypotheses:
+Diffs may appear like this after applying a :tacn:`intro` tactic that results
+in a compacted hypotheses:
 
 ..
 
   .. image:: ../_static/diffs-coqide-compacted.png
      :alt: coqide with Set Diffs on with compacted hypotheses
 
+.. _showing_proof_diffs:
+
+"Show Proof" differences
+````````````````````````
+
+To show differences in the proof term:
+
+- In coqtop and Proof General, use the :cmd:`Show Proof` `Diffs` command.
+
+- In CoqIDE, position the cursor on or just after a tactic to compare the proof term
+  after the tactic with the proof term before the tactic, then select
+  `View / Show Proof with Diffs` from the menu or press `SHIFT-ALT-S` (the default
+  key binding).  Differences will be shown applying the current `Show Diffs` setting
```

Comment:
For the display keys, the default is actually platform-dependent. See `select_arch` in file `preferences.ml`: it is SHIFT-ALT on windows and Linux and SHIFT-COMMAND on macos.

Reply:
I removed the specific key binding.